Specs/addDefaultMatchers.js

```diff
@@ -269,6 +269,22 @@ define([
                 };
             },
 
+            toPickNearbyPrimitive : function(util, customEqualityTesters) {
+                return {
+                    compare : function(actual, expected, config) {
+                        return pickPrimitiveEqualsWrapper(actual, expected, config.x, config.y, config.size);
+                    }
+                };
+            },
+
+            notToPickNearbyPrimitive : function(util, customEqualityTesters) {
+                return {
+                    compare : function(actual, config) {
+                        return pickPrimitiveEqualsWrapper(actual, undefined, config.x, config.y, config.size);
+                    }
+                };
+            },
```

Comment:
In this case it would be fine to send in the values as separate arguments instead of a `config` object.